src/components/Footer.jsx

```diff
@@ -1,5 +1,26 @@
+import whiteLogo from "../../public/assets/whiteLogo.svg";
+import Image from "next/image";
+import { ICONS } from "@/data/icons";
+
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div className="w-full bg-ewb-blue-200 border-t-[20px] border-t-ewb-green pt-32 flex justify-between">
+      <div className="inline-flex pl-6 pt-10 text-white font-bold text-xl">
+        <Image src={whiteLogo} alt="EWB Logo" className="scale-125" />
+        <p className="pt-16 pl-4">EWB at UCR</p>
+      </div>
+
+      <div className="px-1 text-right text-white m-3 text-sm">
+        <div className="ml-3 scale-125 float-right">{ICONS["facebook"]}</div>
```

Comment:
make it a link https://github.com/acm-ucr/ewb-website/blob/dev/src/components/home/Social.jsx